docs/40_smart-contract-guides/80_linking-custom-permission.md

```diff
@@ -2,72 +2,479 @@
+```
+
+The output should be similar to:
+
+```console
+Error 3090005: Irrelevant authority included
+Please remove the unnecessary authority from your action!
+Error Details:
+action declares irrelevant authority '{"actor":"bob","permission":"customp1"}'; minimum authority is {"actor":"bob","permission":"active"}
+```
+
+```shell
+cleos push action scholder what '["name"]' -p bob@customp1
+```
+
+The output should be similar to:
+
+```console
+executed transaction: 43b6ad4ce7a52d7281ccd2800caa02d5278ee714de36fefe9624bff621378402  104 bytes  129 us
+#      scholder <= scholder::what               {"user":"name"}
+>> why not name
+```
+
+```shell
+cleos push action scholder how '["name"]' -p bob@customp1
+```
+The output should be similar to:
+
+```console
+executed transaction: 43b6ad4ce7a52d7281ccd2800caa02d5278ee714de36fefe9624bff621378402  104 bytes  129 us
+#      scholder <= scholder::how               {"user":"name"}
+>> how are you name
+```
+
+We see that the _customp1_ permission can sucessfully call the _what_ and _how_ actions, but fails to call the _why_ action.
+
+Call the actions using the _customp2_ permission:
+
+```shell
+cleos push action scholder why '["name"]' -p bob@customp2
+```
+
+The output should be similar to:
+
+```console
+Error 3090005: Irrelevant authority included
+Please remove the unnecessary authority from your action!
+Error Details:
+action declares irrelevant authority '{"actor":"bob","permission":"customp2"}'; minimum authority is {"actor":"bob","permission":"active"}
+```
+
+```shell
+cleos push action scholder what '["name"]' -p bob@customp2
+```
+
+The output should be similar to:
+
+```console
+Error 3090005: Irrelevant authority included
+Please remove the unnecessary authority from your action!
+Error Details:
+action declares irrelevant authority '{"actor":"bob","permission":"customp2"}'; minimum authority is {"actor":"bob","permission":"customp1"}
+```
+
+```shell
+cleos push action scholder how '["name"]' -p bob@customp2
+```
+
+The output should be similar to:
+
+```shell
+executed transaction: 43b6ad4ce7a52d7281ccd2800caa02d5278ee714de36fefe9624bff621378402  104 bytes  129 us
+#      scholder <= scholder::how               {"user":"name"}
+>> how are you name
+```
+
+We see that the _customp2_ permission can sucessfully call the _how_ action, but fails to call the _why_ and _what_ actions.
+
+## Unlink The Permission
+You can unlink permsssions to a smart contract action.
+
+Now we will unlink the _customp2_ permission from the _how_ action using the command `cleos set action permission`:
+
+```shell
+cleos set action permission bob scholder how NULL -p bob@active
+
+```
+The output should be similar to:
+
+```shell
+executed transaction: 50fe754760a1b8bd0e56f57570290a3f5daa509c090deb54c81a721ee7048201  120 bytes  242 us
+#         eosio <= eosio::unlinkauth            {"account":"bob","code":"scholder","type":"how"}
+
+```
+
+## Test It Again
```

Comment:
suggestion: make this title a subtitle of previous section "Unlink the permission"; why?... I guess it is personal preference, I see it as a subtask of that section and I think it will render nicely on the summary generated on the right side of the page, and emphasise the main steps "Create, Link, Unlink, Delete"  and their subtasks/subtitles.

```
### Test It Again
```

Reply:
Updated